modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/NancyFXServerCodegen.java

```diff
@@ -128,6 +133,10 @@ public void processOpts() {
             setPackageGuid((String) additionalProperties.get(OPTIONAL_PROJECT_GUID));
         }
 
+        if(additionalProperties.containsKey(ASYNC_SERVER)) {
```

Comment:
Please add a space after "if" to conform to Java style guide.

Reply:
done